### PR TITLE
FPCO-13338; Add status command to cli

### PR DESCRIPTION
**Pull request by ManishPrajapati-GoFynd**
Replace all service URL to constants
Added status command related files
Update README.md file

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@gofynd/fdk-cli",
-  "version": "2.4.16",
+  "version": "3.0.0",
```
Comment (vivek8690) [inline package.json:3]: It should be 3.0.1-alpha.1 @manishprajapati52

File: src/commands/status/status-builder.ts [hunk 2]
```diff
@@ -0,0 +1,16 @@
+import commander from 'commander';
+import Status from '../../lib/Status';
+
+export default function statusCommandBuilder() {
+    const status = new commander.Command('status').description('Theme Commands');
+    status
+        .command('check')
+        .alias('verify')
+        .description('Check service status')
+        // .requiredOption('-t, --token [token]', 'Token')
```
Comment (vivek8690) [inline src/commands/status/status-builder.ts:10]: remove comments @manishprajapati52

File: src/lib/api/services/status.service.ts [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+
+        let spinner = ora('Checking services...\n').start();
+
+        const results: PromiseSettledResult<any>[] = await Promise.allSettled(allPrmoises);
+
+        const failedServices = [];
+
+        results.forEach(result => {
+            if (result.status === 'fulfilled') {
+                // console.log(chalk.green('\u2713 ' + result.value.config.data));
```
Comment (vivek8690) [inline src/lib/api/services/status.service.ts:27]: remove console log

File: src/lib/api/services/utils.ts [hunk 4]
```diff
@@ -1,10 +1,17 @@
...
             'Content-Type': 'application/json',
             'x-fp-cli': `${packageJSON.version}`,
-        }
+        },
     };
-};
\ No newline at end of file
+};
+
+export const getBaseURL = () => {
```
Comment (vivek8690) [inline src/lib/api/services/utils.ts:14]: why this method is needed ? @manishprajapati52

**Review by vivek8690 — CHANGES_REQUESTED** [review]

File: src/lib/api/services/utils.ts (hunk 4 above)
Comment (ManishPrajapati-GoFynd) [reply src/lib/api/services/utils.ts:14]: Before we had getBaseURL method in `lib>api>services>url.ts` and was using in one file only. But now, as we are using this method in multiple files, I moved this method to utils.

**Comment by ManishPrajapati-GoFynd** [comment]
Duplicate: https://github.com/gofynd/fdk-cli/pull/200

**Pull request closed without merging**